supabase/seed.sql

```diff
@@ -0,0 +1,6 @@
+insert into meetups
```

Comment:
@MildTomato sorry this isn't really needed

Reply:
I think it's good. can leave it in.